core/App/components/listItems/NotificationListItem.tsx

```diff
@@ -93,6 +94,15 @@ const NotificationListItem: React.FC<NotificationListItemProps> = ({ notificatio
           .getParent()
           ?.navigate(Stacks.NotificationStack, { screen: Screens.ProofRequest, params: { proofId: notification.id } })
       break
+      case NotificationType.Revocation:
+      title = t('CredentialDetails.NewRevoked')
```

Comment:
Seems like the formatting is off here. But this PR is running into a number of linter issues

Reply:
Yeah sorry for the linting errors, I've had issues running the linter or formatter locally ever since we switched to the new project structure. These should all be resolved now